app/lib/actions/katello/host/update.rb

```diff
@@ -0,0 +1,40 @@
+module Actions
+  module Katello
+    module Host
+      class Update < Actions::EntryAction
+        middleware.use ::Actions::Middleware::RemoteAction
+
+        def plan(host, consumer_params = nil)
+          action_subject host
+          host.content_aspect.save!
+          host.subscription_aspect.save!
+
+          if consumer_params
+            host.subscription_aspect.update_from_consumer_attributes(consumer_params)
+            host.subscription_aspect.save!
+          else
+            consumer_params = host.subscription_aspect.consumer_attributes
+            host.content_host.content_view = host.content_view
+            host.content_host.environment = host.lifecycle_environment
+            host.content_host.save!
```

Comment:
The content_aspect's content_view and lifecycle_environment don't require updating?